frontend/components/commons/sidebar/SidebarPanel.vue

```diff
@@ -16,12 +16,14 @@
   -->
 
 <template>
-  <aside
-    :style="{ top: topPosition }"
-    :class="['sidebar', annotationEnabled ? 'annotation' : 'explore']"
-  >
+  <aside :class="['sidebar', annotationEnabled ? 'annotation' : 'explore']">
```

Comment:
Define rule and explore shares class ?

Reply:
@frascuchon we can remove mode reference with the new header